ansible/roles/cassandra-db-update/templates/logback.j2

```diff
@@ -0,0 +1,149 @@
+            <appender-ref ref="ASYNC_TRANSACTION_HANDLER"/>
+        </logger>
+        <!-- content_data transaction logs -->
+        <appender name="TRANSACTION_HANDLER_CONTENT_DATA" class="ch.qos.logback.core.rolling.RollingFileAppender">
+            <file>${cassandra.logdir}/cassandra-triggers/content_data_transactions.log</file>
+            <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+                <fileNamePattern>${cassandra.logdir}/cassandra-triggers/content_data_transactions-${timestamp}-%i.log</fileNamePattern>
+                <minIndex>1</minIndex>
+                <maxIndex>20</maxIndex>
+            </rollingPolicy>
+            <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
+                <maxFileSize>5MB</maxFileSize>
+            </triggeringPolicy>
+            <encoder>
+                <pattern>%d %msg%n</pattern>
+            </encoder>
+        </appender>
+        <appender name="ASYNC_TRANSACTION_HANDLER_CONTENT_DATA" class="ch.qos.logback.classic.AsyncAppender">
+            <queueSize>1024</queueSize>
+            <discardingThreshold>0</discardingThreshold>
+            <includeCallerData>true</includeCallerData>
+            <appender-ref ref="TRANSACTION_HANDLER_CONTENT_DATA" />
+        </appender>
+        <logger name="org.sunbird.cassandra.triggers.{{ content_keyspace_name }}.content_data" level="INFO" additivity="false">
+            <appender-ref ref="ASYNC_TRANSACTION_HANDLER_CONTENT_DATA"/>
+        </logger>
+        <!-- question data transaction logs -->
+        <appender name="TRANSACTION_HANDLER_QUESTION_DATA" class="ch.qos.logback.core.rolling.RollingFileAppender">
+            <file>${cassandra.logdir}/cassandra-triggers/question_data_transactions.log</file>
+            <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+                <fileNamePattern>${cassandra.logdir}/cassandra-triggers/question_data_transactions-${timestamp}-%i.log</fileNamePattern>
+                <minIndex>1</minIndex>
+                <maxIndex>20</maxIndex>
+            </rollingPolicy>
+            <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
+                <maxFileSize>5MB</maxFileSize>
+            </triggeringPolicy>
+            <encoder>
+                <pattern>%d %msg%n</pattern>
+            </encoder>
+        </appender>
+        <appender name="ASYNC_TRANSACTION_HANDLER_QUESTION_DATA" class="ch.qos.logback.classic.AsyncAppender">
+            <queueSize>1024</queueSize>
+            <discardingThreshold>0</discardingThreshold>
+            <includeCallerData>true</includeCallerData>
+            <appender-ref ref="TRANSACTION_HANDLER_QUESTION_DATA" />
+        </appender>
+        <logger name="org.sunbird.cassandra.triggers.{{ content_keyspace_name }}.question_data" level="INFO" additivity="false">
```

Comment:
@gandham-santhosh : will content_keyspace_name get resolved during copy from xml also? If not, please let me know the correct way to do it.

Reply:
Yes. It will get resolved